packages/utils/src/lib/filter-by-slug.ts

```diff
@@ -0,0 +1,39 @@
+import { Audit, Group } from '@code-pushup/models';
+import { toArray } from './transform';
+
+export function filterByAuditSlug(
+  groups: Group[],
+  auditSlugs: string | string[],
+): Group[] {
+  const slugs = toArray(auditSlugs);
+  if (slugs.length === 0) {
+    return groups;
+  }
+  return (
+    groups
+      .map(group => ({
+        ...group,
+        refs: filterSlug(group.refs, slugs),
+      }))
+      // filter out groups that have no audits includes from onlyAudits (avoid empty groups)
+      .filter(group => group.refs.length)
+  );
+}
+
+export function filterBySlug(
```

Comment:
It would be helpful to distinguish the name of these functions based on what the arrays contain.
```suggestion
export function filterAuditsBySlug(
```